ledger/src/shred.rs

```diff
@@ -127,6 +127,17 @@ bitflags! {
     }
 }
 
+impl ShredFlags {
+    /// Creates a new ShredFlags from the given reference_tick
+    ///
+    /// SHRED_TICK_REFERENCE_MASK is comprised of only six bits whereas the
+    /// reference_tick has 8 bits (u8). The reference_tick bits will saturate
+    /// in the event that reference_tick > SHRED_TICK_REFERENCE_MASK
+    pub(crate) fn new_from_reference_tick(reference_tick: u8) -> Self {
```

Comment:
slight preference to just name this `from_...` (i.e. "`new_`" is redundant).
similar to `From` trait: https://doc.rust-lang.org/std/convert/trait.From.html

(not suggesting to implement `From<u8>` though).